content/news/014.general_meeting_2024.en.md

```diff
@@ -0,0 +1,11 @@
+---
+published: 2024-12-05
+description: We reflect on an eventfull year
+title: DD-IX Generalassembly 2024
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 014_general_assembly_24.webp
+---
+
+The general meeting of an association is an important event every year to bring your members together, foster new ideas and report on our activities. We held our general meeting on December 2th and welcomed our member at the Department of Computer Science at TU Dresden. The DD-IX board reported on the finances, developments and events of the past year and gave an outlook for the coming year. So be prepared for another year of exciting developments from DD-IX.
```

Comment:
[...] to bring members together, foster new ideas, and report on activities. We held our first general meeting on December 2, 2024 at the Faculty of Computer Science at TU Dresden. The DD-IX board reported on the finances, achievements, and events of the past year and gave an outlook for the 2025. Be prepared for another year of exciting developments of DD-IX.